airbyte-integrations/connectors/source-rest-api/airbyte_protocol

```diff
@@ -0,0 +1 @@
+/Users/charles/code/airbyte/airbyte-integrations/bases/airbyte-protocol/airbyte_protocol
```

Comment:
use relative path

Reply:
oh interesting. this was autogenerated. but yes. definitely relative path. will try to figure out why template is spitting out absolute path.

Reply:
i fixed this in the template https://github.com/airbytehq/airbyte/pull/783. will fix here now.